### PR TITLE
Feats

### PR DESCRIPTION
**Pull request by abias1122**
## What does this do?
Adds feats (well, just Grappler) to the REST and graphql APIs

## How was it tested?
I added unit tests and integration tests and ran them. I also hit the new endpoint locally using Postman.

## Is there a Github issue this is resolving?
No

### CODE REVIEW

File: src/views/partials/doc-content.ejs [hunk 1]
```diff
@@ -151,6 +149,9 @@
   <h1 id="spell-section">Spells</h1>
   <% include ../partials/doc-resource-spells %>
 
+  <h1 id="feats-section">Feats</h1>
```
Comment (bagelbits) [inline src/views/partials/doc-content.ejs:152]: You'll also want to add to the doc-left-col.ejs file so it shows up in the side menu.

**Review by bagelbits — COMMENTED** [review]
Just one comment on adding to the left column partial for the docs, but everything else looks great!

File: src/views/partials/doc-content.ejs (hunk 1 above)
Comment (abias1122) [reply src/views/partials/doc-content.ejs:152]: Done.

**Review by bagelbits — APPROVED** [review]
LGTM!

**Pull request merged**